haystack/modeling/data_handler/data_silo.py

```diff
@@ -136,61 +113,21 @@ def _get_dataset(self, filename: Optional[Union[str, Path]], dicts: Optional[Lis
                         random.shuffle(dicts)
 
         num_dicts = len(dicts)
-        multiprocessing_chunk_size, num_cpus_used = calc_chunksize(
-            num_dicts=num_dicts, max_processes=self.max_processes, max_chunksize=self.max_multiprocessing_chunksize
-        )
-
-        with ExitStack() as stack:
-            if self.max_processes > 1:  # use multiprocessing only when max_processes > 1
-                if self.multiprocessing_strategy:
-                    if self.multiprocessing_strategy in mp.get_all_sharing_strategies():
-                        mp.set_sharing_strategy(self.multiprocessing_strategy)
-                    else:
-                        logger.warning(
-                            f"{self.multiprocessing_strategy} is unavailable, "
-                            f"falling back to default multiprocessing sharing strategy of your OS."
-                        )
-
-                p = stack.enter_context(mp.Pool(processes=num_cpus_used))
-
-                logger.info(
-                    f"Got ya {num_cpus_used} parallel workers to convert {num_dicts} dictionaries "
-                    f"to pytorch datasets (chunksize = {multiprocessing_chunk_size})..."
-                )
-                log_ascii_workers(num_cpus_used, logger)
```

Comment:
In case we remove multiprocessing from inference as well in a separate PR, then we could remove the implementation of ` log_ascii_workers` completely from [haystack/modeling/utils.py](https://github.com/deepset-ai/haystack/blob/main/haystack/modeling/utils.py)